andela/zinnia-ah-frontend
# 164797109 user can login

#### What does this PR do?
* Implement login feature in the app

#### Description of Task to be completed?
* Write tests, including tests for components and integration tests
* Create components for login
* Add authentication to input fields
* Connect with redux

#### How should this be manually tested?
* Clone the repository and start server
* Visit the url, `localhost:8080/login`
* A page containing the login form should be seen

#### Any background context you want to provide?
* None

#### What are the relevant pivotal tracker stories? (if applicable)
https://www.pivotaltracker.com/story/show/164797109

#### Screenshots
<img width="1440" alt="Screenshot 2019-05-13 at 19 31 25" src="https://user-images.githubusercontent.com/15995472/57645333-f31ff900-75b5-11e9-9ad9-ba5b41542b7f.png">
<img width="1440" alt="Screenshot 2019-05-13 at 19 31 59" src="https://user-images.githubusercontent.com/15995472/57645334-f31ff900-75b5-11e9-9df8-6e77e0f5b065.png">
